### ISSUE TITLE
Don't show line in poles

### ISSUE THREAD

**Issue by piranna** [nuclearsecrecy/Leaflet.greatCircle#3]
I understand that the Great Circle is drawn using polygons, and once it gets to the North or South pole is just adjust the points to be fixed there... but it's a bit of annoying. I would like that for the polygons border lines that are in the poles they would have no border, so it looks better it's a "continuation" of the cover area instead of a "limit" frontier of the Great Circle itself.

**Comment by nuclearsecrecy** [comment]
I agree that it's annoying! If I had an easy way to get rid of it, I would. But what I recall finding was that even if I tried to draw the lines as separate lines that didn't connect, they ended up looking weird because they didn't look like they were really continuous — they looked weirder than the obviously-weird polar line. But it has been a long time since I've played with this code, and I might be giving it another look this summer, and see if I can improve it.

**Comment by piranna** [comment]
Maybe just drawing the polygon without the borders would be enough, or paint later a line over the border, but that would increase CPU usage, so that would need to be done only if the current bordered polygon is giving the polar issue.
